packages/renderer/src/lib/pod/PodsList.svelte

```diff
@@ -126,7 +127,15 @@ async function deleteSelectedPods() {
 }
 
 function openDetailsPod(pod: PodInfoUI) {
-  router.goto(`/pods/${encodeURI(pod.kind)}/${encodeURI(pod.name)}/${encodeURI(pod.engineId)}/logs`);
+  const route = PodDetailsRoute.getRoute(
```

Comment:
we can ask podUtils object instead of adding a new PodDetailsRoute

also we can provide podInfoUi object to the method instead of providing each individual parameter

Reply:
I would not want to make the (generic) Route methods coupled with the podInfoUi. For example in the following lines, it would need to build such an podInfoUi object from the ContainerGroupInfoUI object and the `podman` constant:
https://github.com/containers/podman-desktop/blob/main/packages/renderer/src/lib/ContainerList.svelte#L349-L351

I would prefer to simply call it as:
```
PodDetailsRoute.getRoute({
  kind: 'podman',
  name: containerGroup.name,
  engineId: containerGroup.engineId
})
```

Reply:
but you can have two methods, one with PodInfoUI and one with meta.params so it's convenient on client side

Reply:
in the example you shown you can provide containerGrou/ContainerGroupInfoUI object instead of building a new object